public-ui/src/app/data-browser/chart/chart.component.ts

```diff
@@ -81,11 +81,22 @@ export class ChartComponent implements OnChanges {
         pie: {
           borderColor: null,
           slicedOffset: 4,
+          size: this.analysis ?
+              (this.analysis.analysisId === this.dbc.SURVEY_GENDER_IDENTITY_ANALYSIS_ID ||
```

Comment:
For these 3 cases I would probably pull this boolean (or two booleans?) out into a descriptively named local varaible, I'm not following what you are checking for here:

```
const isSurveyGenderAnalysis = ...;
...
size: isSurveryGenderAnalysis ? '60%' : '100%';
```

Reply:
The gender charts need a toggle button on the top which is making the gender chart overflow in to age so was trying to reduce the height in that case.